ext/debug/ctf/README.rst

```diff
@@ -0,0 +1,204 @@
+.. _ctf:
```

Comment:
Note: we don't include any content from /ext into our published documentation at http://docs.zephyrproject.org.  If the intention is to have this material show up in our documentation, we'll need to put it somewhere in a /doc subfolder